src/main/java/com/es/phoneshop/model/history/ProductHistoryList.java

```diff
@@ -3,6 +3,7 @@
 import com.es.phoneshop.model.product.Product;
 
 import java.util.ArrayList;
+import java.util.LinkedList;
 import java.util.List;
 
 public class ProductHistoryList {
```

Comment:
Change the name to smth that doesn't end with List otherwise it seems to be List implementation